src/homepage/_posts/2020-04-06-james-3.5.0.markdown

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title:  "Apache James Server 3.5.0"
+date:   2020-04-06  15:16:30 +0200
+categories: james update
+---
+
+The Apache James developers are pleased to announce James server 3.5.0 release.
+
+Here are some points we worked on:
+
+### Added
+- Distributed task management for Guice cassandra-rabbitmq product. This enables several James servers to share a consistent view
+of tasks being currently executed.
+- JAMES-2563 Health check for ElasticSearch
+- JAMES-2904 Authentication and SSL support for Cassandra backend
+- JAMES-2904 Authentication and SSL support for ElasticSearch backend
+- JAMES-3066 Add "allowed From headers" webadmin endpoint
+- JAMES-3062 EventDeadLettersHealthCheck
+- JAMES-3058 WebAdmin offline task to correct mailbox inconsistencies on top of Cassandra products
```

Comment:
Could we group tickets by product?

Reply:
They all concern "Distributed James guice.

However the question is interesting: should we have a changelog per product?

I fear this might be overwhelming for contributors to maintain such a thing...

Reply:
right now, the changelog mixes things that are not related.
I just propose in each section (Added, Changed, Fixed) to have sub-sections like: Distributed, JPA, Core, Admin

Reply:
Do we duplicate changes that concerns multiple products?

Reply:
If it's something late in the release process we could address it in the next release to avoid blocking this one.

Reply:
> If it's something late in the release process we could address it in the next release to avoid blocking this one.

:+1: and we should fill somewhere on the contributing guidelines how to fill the changelog.